packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -142,6 +144,13 @@ export const ProjectMigratorMW: Middleware = async (ctx: CoreHookContext, next:
   await next();
 };
 
+async function checkMethod(ctx: CoreHookContext) {
+  const getProjectConfigMethod = "getProjectConfig";
+  if (ctx.method === getProjectConfigMethod && fromReloadFlag) return false;
```

Comment:
Why not just delete `ProjectMigratorMW` for `getProjectConfig` method in `fx-core/src/core/index.ts`

Reply:
If delete, dialog will not be popped up when user open project.